<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright (C) 2005 Peter Simonsson</author>
 <comment>This program is free software; you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation; either version 2 of the License, or
(at your option) any later version.
</comment>
 <class>ConnectionBehavior_Config</class>
 <widget class="QWidget" name="ConnectionBehavior_Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>432</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="kcfg_AutoReconnect">
     <property name="title">
      <string>Enable Automatic Reconnect</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="reconnectDelayLabel">
        <property name="text">
         <string>Reconnect delay:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>kcfg_ReconnectDelay</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Reconnection attempts:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KPluralHandlingSpinBox" name="kcfg_ReconnectDelay">
        <property name="value">
         <number>120</number>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="kcfg_ReconnectCount">
        <property name="value">
         <number>100</number>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="specialValueText">
         <string>Unlimited</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="debugGBox">
     <property name="title">
      <string>Debug</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="kcfg_RawLog">
        <property name="text">
         <string>Show raw &amp;log window when connecting</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="encryptionGBox">
     <property name="title">
      <string>Encryption</string>
     </property>
     <layout class="QFormLayout" name="encryptionLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Default encryption type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="kcfg_EncryptionType">
        <property name="whatsThis">
         <string>ECB or Electronic Codebook mode is supported by many encryption clients like FiSH and Mircryption however it is generally considered to be insecure.

CBC or Cipher Block Chaining mode is not supported by FiSH but is supported by Mircryption. If you are going to use key exchange it is best to first find out if your peer has a client that can use CBC otherwise you will be able to see what they are saying, but they will not be able to read what you are saying.</string>
        </property>
        <item>
         <property name="text">
          <string>Electronic Codebook (ECB)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cipher-Block Chaining (CBC)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="kcfg_ProxyEnabled">
     <property name="title">
      <string>Proxy</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="proxyLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="typeLabel">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="kcfg_ProxyType">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Socks v5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HTTP</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="addressLabel">
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="kcfg_ProxyAddress">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="kcfg_ProxyPort">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="KLineEdit" name="kcfg_ProxyUsername">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="KPasswordLineEdit" name="m_ProxyPassword">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPasswordLineEdit</class>
   <extends>QWidget</extends>
   <header>kpasswordlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPluralHandlingSpinBox</class>
   <extends>QSpinBox</extends>
   <header>kpluralhandlingspinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>kcfg_AutoReconnect</tabstop>
  <tabstop>kcfg_ReconnectDelay</tabstop>
  <tabstop>kcfg_ReconnectCount</tabstop>
  <tabstop>kcfg_RawLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
